RandomForestClassifier seems to fail when used as a standalone model

The `test_random_forest_classifier_double_features` test surfaced that there are cases where `RandomForestClassifier` returns wrong predictions. It seems related to it being used as a standalone model outside of a full pipeline.   

```
E           SQL and sklearn predictions don't match
E           Mismatched elements: 2 / 8 (25%)
E           Max absolute difference among violations: 1
E           Max relative difference among violations: 1.
E            ACTUAL: array([0, 1, 0, 1, 0, 1, 0, 1])
E            DESIRED: array([1, 1, 0, 1, 0, 1, 1, 1])
```

--- a/src/orbital/translation/steps/trees/classifier.py
+++ b/src/orbital/translation/steps/trees/classifier.py
@@ -166,9 +166,18 @@ class TreeEnsembleClassifierTranslator(Translator):
                     apply_post_transform(total_score, post_transform),
                 )
 
+            # scikit-learn resolves a probability tie via ``argmax``, which
+            # returns the lowest class index (class 0). The ONNX leaf weights
+            # are stored as float32 (e.g. ``1 / n_estimators`` becomes
+            # ``0.10000000149...``), so a vote split that should sum to exactly
+            # 0.5 comes out marginally above it once promoted to double and
+            # would otherwise flip the tie to class 1. Requiring the score to
+            # exceed 0.5 by a small epsilon breaks ties toward class 0 to match
+            # scikit-learn while staying well below any genuine probability gap.
+            tie_epsilon = 1e-6
             label_expr = optimizer.fold_case(
                 ibis.cases(
-                    (total_score > 0.5, output_classlabels[1]),
+                    (total_score > 0.5 + tie_epsilon, output_classlabels[1]),
                     else_=output_classlabels[0],
                 )
             )

--- a/src/orbital/translation/steps/trees/classifier.py
+++ b/src/orbital/translation/steps/trees/classifier.py
@@ -30,6 +30,10 @@ class TreeEnsembleClassifierTranslator(Translator):
     the votes by the sum of all votes.
     """
 
+    # Margin by which a binary probability must exceed 0.5 to be counted as
+    # a win for class 1, used to break exact 0.5 ties toward class 0.
+    TIE_EPSILON = 1e-6
+
     def process(self) -> None:
         """Performs the translation and set the output variable."""
         # https://onnx.ai/onnx/operators/onnx_aionnxml_TreeEnsembleClassifier.html
@@ -174,10 +178,9 @@ class TreeEnsembleClassifierTranslator(Translator):
             # would otherwise flip the tie to class 1. Requiring the score to
             # exceed 0.5 by a small epsilon breaks ties toward class 0 to match
             # scikit-learn while staying well below any genuine probability gap.
-            tie_epsilon = 1e-6
             label_expr = optimizer.fold_case(
                 ibis.cases(
-                    (total_score > 0.5 + tie_epsilon, output_classlabels[1]),
+                    (total_score > 0.5 + self.TIE_EPSILON, output_classlabels[1]),
                     else_=output_classlabels[0],
                 )
             )

--- a/tests/test_models.py
+++ b/tests/test_models.py
@@ -278,9 +278,6 @@ class TestSingleStepPipelines:
             separate_trees=separate_trees,
         )
 
-    @pytest.mark.skip(
-        reason="SQL and sklearn predictions don't match - needs investigation"
-    )
     def test_random_forest_classifier_double_features(self):
         """Test RandomForestClassifier with all double features."""
         features = {

